src/dhcp_fsm.erl

```diff
@@ -156,10 +156,10 @@ offered(Pkg = #dhcp_package{xid = _XId, message_type = request},
             case delegate(request, Pkg, State) of
                 {ok, RPkg, State1} ->
                     Timeout = dhcp_package:get_option(ip_address_lease_time, dhcp_package:get_option(ip_address_lease_time, 3000, RPkg), Pkg),
-                    {next_state, bound, State#state{last=erlang:now(),
+                    {next_state, bound, State#state{last=current_time(),
                                                     handler_state = State1}, ?S(Timeout)};
                 {ok, State1} ->
-                    {next_state, offered, State#state{last=erlang:now(),
+                    {next_state, offered, State#state{last=current_time(),
                                                       handler_state = State1}, ?S(To)};
```

Comment:
Line 163 is too long:                                                       handler_state = State1}, ?S(To)};.